onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -114,6 +116,15 @@ contract CartesiDApp is
         _consensus.join();
     }
 
+    function supportsInterface(
+        bytes4 interfaceId
+    ) public view virtual override(ERC1155Receiver, IERC165) returns (bool) {
```

Comment:
if we upgrade openzeppelin, this function would be overriding `ERC1155Holder` rather than `ERC1155Receiver`